lib/mpi/constants.rb

```diff
@@ -20,10 +20,12 @@ module Constants
     IDME_IDENTIFIER = '200VIDM'
     LOGINGOV_IDENTIFIER = '200VLGN'
     DSLOGON_IDENTIFIER = '200DOD'
+    VHA_IDENTIFIER = 'USVHA'
 
     IDME_FULL_IDENTIFIER = 'PN^200VIDM^USDVA^A'
     LOGINGOV_FULL_IDENTIFIER = 'PN^200VLGN^USDVA^A'
     DSLOGON_FULL_IDENTIFIER = 'NI^200DOD^USDOD^A'
+    VHA_FULL_IDENTIFIER = 'USVHA^A'
```

Comment:
I prob wouldn't call this a 'full identifier', I would probably call this `ACTIVE_VHA_IDENTIFIER`